app/main/helpers/services.py

```diff
@@ -241,7 +241,7 @@ def _is_pricing_type(key):
 
 def _is_type(key, *types):
     """Return True if a given key is one of the provided types"""
-    return new_service_content.get_question(key)['type'] in types
+    return new_service_content.get_question(key).get('type') in types
```

Comment:
I thought `serviceTypes` was handled by `_is_list_type` explicitly, and it's the first check in `get_formatted_section_data`, where is the error triggered from?